content/20_CI_CD_Pipeline/_index.md

```diff
@@ -1,25 +1,81 @@
 
-### Continuous Development (CD) and Continuous Deployment (CD)
+AWS brings in a complete set of CI/CD developer tools to accelerate software development and release cycles. AWS CodePipeline automates the build, test, and deploy phases of the release process every time there is a code change, based on the defined release model. This enables the rapid and reliable delivery of features and updates.
+AWS CodePipeline can address a variety of development and operation use cases including:
 
-Continuous Development (CD) is the process of automatically deploying all code changes to a testing and/or production environment after the build stage of your workflow is complete. This means that on top of automated testing, you have an automated release process and you can deploy your application any time by clicking a button. You have the option to decide how often you want to deploy your code but we recommend that you deploy your code to production in small batches as early as possible in cases where you need to troubleshoot. 
+    - Compiling, building, and testing code with AWS CodeBuild.
+    - Continuous delivery of container-based applications to the cloud.
+    - Pre-deployment validation of artifacts (such as descriptors and container images) 
+      required for network service or specific cloud-native network functions.
+    - Functional, integration, and performance tests, including baseline and 
+      regression testing.
+    - Reliability and disaster recovery (DR) testing.
 
-CD can also be known as continuous deployment which is different than continuous development as defined above. Continuous deployment actually goes a step further than continuous development in that every change that passes all stages of your production pipeline is released to your end users or customers. There is no human intervention and only a failed test will prevent a new change to be deployed to production. The benefit here is that you accelerate deployment times and your developer teams can focus on building software as they can see their work go live instantenously. 
 
-In the following sections, we will introduce you to AWS CodeBuild and AWS CodePipeline along with setting up the different AWS resources we will be using to use our pipeline.
+AWS CodePipeline can integrate with other services. These can be AWS services such as Amazon Simple Storage Service (Amazon S3), or third-party products such as GitHub, Docker BuildKit, Docker Scout and Docker Hub.
+
+![Docker](/images/cicdpipeline2.png)
```

Comment:
could we add a better substitued name so instead of `Docker` we can use `CodePipeline-2`